exporter/loggingexporter/factory.go

```diff
@@ -43,28 +44,53 @@ func (f *Factory) CreateDefaultConfig() configmodels.Exporter {
 			TypeVal: typeStr,
 			NameVal: typeStr,
 		},
+		LogLevel: "info",
```

Comment:
It is natural to make the default log level "info" but then some configurations that we already have around will probably make this exporter goes silent. Could you please also change the the exporter current message to be at Info level? Later we can add more information, e.g.: serviceName etc on a verbose log.

Reply:
IIUC, the current log level used in the logging exporter is `debug` level [1]. So do you suggest changing that to `logger.Info(`? Thanks.

[1] https://github.com/open-telemetry/opentelemetry-service/blob/52d022665aae92638cd0ae7844cea551107f4cc9/exporter/loggingexporter/logging_exporter.go#L47

Reply:
It is that now with the old configurations one ends up getting debug log for the service stuff but then the logging exporter is going to stop logging data because its default is info, makes sense? For example: examples/demo/

Reply:
It seems to me that these summarized log message could be in the info level as the default setting.

Reply:
Exactly

Reply:
Thanks. If you agree with the new commit, I will squash them into a single one.